### PR TITLE
adding missing snapshot auth credentials

### PR DESCRIPTION
**Pull request by sockeqwe**
I have noticed that currently travis can't push snapshots because auth credentials are missing (http status code 401).

I have added them to maven push script.

All you have to do is merge this request and then setup the auth credentials with the following commands:
```
travis encrypt -r bluelinelabs/Conductor "NEXUS_USERNAME=InsertYourUsername" --add
travis encrypt -r bluelinelabs/Conductor "NEXUS_PASSWORD=InsertYourPassword" --add
```

### CODE REVIEW

File: gradle-mvn-push.gradle [hunk 1]
```diff
@@ -32,11 +32,21 @@ def getSnapshotRepositoryUrl() {
 }
 
 def getRepositoryUsername() {
-  return hasProperty('NEXUS_USERNAME') ? NEXUS_USERNAME : ""
+  def username = "$System.env.NEXUS_USERNAME"
+  if (username == null) {
```
Comment (JakeWharton) [inline gradle-mvn-push.gradle]: This will never be null

File: gradle-mvn-push.gradle [hunk 2]
```diff
@@ -32,11 +32,21 @@ def getSnapshotRepositoryUrl() {
 }
 
 def getRepositoryUsername() {
-  return hasProperty('NEXUS_USERNAME') ? NEXUS_USERNAME : ""
```
Comment (JakeWharton) [inline gradle-mvn-push.gradle]: You can use `ORG_GRADLE_PROJECT_NEXUS_USERNAME` to set properties from environment variables and retain the original behavior.

Comment (sockeqwe) [inline gradle-mvn-push.gradle]: Ah, didn't know about that, thanks!

So basically, this pulll request can be rejected and @EricKuck only has to set:

````
travis encrypt -r bluelinelabs/Conductor "ORG_GRADLE_PROJECT_NEXUS_USERNAME=InsertYourUsername" --add
travis encrypt -r bluelinelabs/Conductor "ORG_GRADLE_PROJECT_NEXUS_PASSWORD=InsertYourPassword" --add
```

right?

Comment (JakeWharton) [inline gradle-mvn-push.gradle]: Yep!

**Pull request closed without merging**